src/vaadin-grid-pro-inline-editing-mixin.html

```diff
@@ -131,7 +142,9 @@
         });
       } else {
         this.addEventListener('dblclick', e => {
-          this._enterEditFromEvent(e);
+          if (!this.editOnClick) {
```

Comment:
Had to surround this with a check, because I noticed that on columns with a select, the overlay would be incorrect opened and the cell could stay on an invalid edition state if, on the colum with `editor-type="select"`:
- click on a cell
- double-click (it worked even with two separate clicks) on another cell of the same column

The first overlay then moves to the top-left of the page.

I couldn't figure out a way to test it programmatically, though.